docs/developer/README.md

```diff
@@ -4,10 +4,23 @@ title: 'API and client library documentation'
 
 # Developer
 
-This section helps developers build applications that use drand as a source of randomness. For help running a drand network, see the [Operator's Guide](/operator/).
+This section helps developers build applications that use drand as a source of randomness. For help running your own drand network, see the [Operator's Guide](/operator/).
 
-There are several ways to consume randomness from a drand network:
+The current public League of Entropy drand mainnet API endpoints are:
 
-- [Fetching randomness over HTTP](/developer/http-api/)
-- [Using a drand client library](/developer/clients/)
-- [Using the `drand` binary](/developer/cli/)
+* `http(s)://api.drand.sh` (HTTP Endpoint)
+* `http(s)://api2.drand.sh` (HTTP Endpoint)
+* `http(s)://api3.drand.sh` (HTTP Endpoint)
```

Comment:
`(s)` in the URL makes it invalid and so kinda annoying to copy and paste - I'd separate them or just list the https one.

Reply:
Agreed. If the `https` one works, let's keep that :)

Reply:
People should only use HTTPS whenever they have no other choice as it is slower and costs us more so I want to make it an option at least.